src/xcode/ENA/ENA/Source/Services/__tests__/Mocks/MockTestStore.swift

```diff
@@ -90,6 +90,10 @@ final class MockTestStore: Store, PPAnalyticsData {
 	var previousPpacApiTokenSrs: TimestampedToken?
 	var otpTokenSrs: OTPToken?
     var otpSrsAuthorizationDate: Date?
+	/// DEPRECATED, Only used for dev menu testing for API tokens
+	var ppacApiTokenEdus: TimestampedToken?
+	/// DEPRECATED, Only used for dev menu testing for API tokens
+	var ppacApiTokenEls: TimestampedToken?
```

Comment:
```suggestion
	@available(*, deprecated, message: "Only used for dev menu testing for API tokens")
	var ppacApiTokenEdus: TimestampedToken?
	@available(*, deprecated, message: "Only used for dev menu testing for API tokens")
	var ppacApiTokenEls: TimestampedToken?
```

Reply:
but is can still be accessed after using this ?

Reply:
Yes, that should leave a warning message. But it's not important, you can also keep it as it is.